mycore-base/src/main/java/org/mycore/common/xml/MCRRateLimitBuckets.java

```diff
@@ -0,0 +1,117 @@
+package org.mycore.common.xml;
```

Comment:
Minor quibble: `MCRRateLimitBuckets` should not be in `org.mycore.common.xml` but in `org.mycore.common` as it could possibly be used in places other than the URI resolver.